testsuite/run_forge.sh

```diff
@@ -9,6 +9,18 @@
 # if the necessary environment variables are set.
 #
 
+# Calculate a shard for cutting over cleanly
+SHARD_KEY="${GITHUB_USER:-$USER}"
+SHARD="$(echo $SHARD_KEY | PYTHONHASHSEED=1 python3 -c "print(abs(hash(input())) % 100)")"
+MAX_SHARD="${MAX_SHARD:-0}"
```

Comment:
where are you defining `MAX_SHARD`? do you mean to have it default to 0 at first?

Reply:
This allows it to be passed via env var or default to 0 which is off